Connect for every dot all possibles lines to another dot

cfguimaraes opened the pull request:
Made a refactoring in methods to extract common code for centralized method

cfguimaraes committed:
Connect for every dot all possibles lines to another dot

Made a refactoring in methods to extract common code for centralized method

mpinheir commented:
Obrigado pela contribuição. Ajusta por favor para o procedimento poder lidar com qualquer quantia de pontos e não apenas 100 pontos. Talvez o ideal seja permitir o usuário entrar a quantidade de pontos a ser usada pela rotina. Faz a alteração por favor e manda o commit que o Pull Request vai ser atualizado automagicamente.

cfguimaraes commented:
Estava até pensando nisso mesmo, mas como ví que o seu código sempre levava em consideração a quantidade fixa não quis fazer um único fluxo diferente e passar com o carro na frente dos bois.
Assim que puder faça essa pequena alteração.
O que achou do *refactoring*?

cfguimaraes committed:
Methods reordened to follow sequences os call in html.
Added code to remember last function called to reexecute when apply function are called.

cfguimaraes commented:
Adicionei um código para permitir que seja inputado a quantidade de pontos a serem selecionados.
Também reordenei um método para seguir a mesma sequencia do HTML.

mpinheir commented:
Ok. Vou fazer o merge e vamos ver como vai ficar.
